R/forecast_time_series.R

```diff
@@ -315,7 +324,7 @@ forecast_time_series <- function(input_data,
   }
   
   # parallel run within azure batch
-  if(parallel_processing=="azure_batch") {
+  if(sum(parallel_processing=="azure_batch") == 1) {
```

Comment:
Thank you @Mike, Please fix this as discussed.

Reply:
updated PR based on comments